src/components/Input.tsx

```diff
@@ -5,16 +5,23 @@ const Input = ({
   placeholder,
   helpText,
   disabled,
+  value,
+  onChange,
+  required,
 }: {
   type: string;
   name: string;
   label: string;
   placeholder?: string;
   helpText?: string;
   disabled?: boolean;
+  className?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  required?: string;
```

Comment:
shouldn't required be a boolean value?

Reply:
Here I am taking `required` as a `string` and checking directly in line number:- 39 that if `required` data is there then show the error in required else skip it.